### DISCUSSION TITLE
Terminal Music Player

### DISCUSSION THREAD

**Discussion by TrevorSatori** [fdehau/tui-rs#691] [Show and tell]
I love terminal apps and wanted to create a rewrite of something like cmus in Rust. This framework not only allowed that to happen, but be done with ease. The primary difference between this and spotify-tui, is this is meant to be used for local audio files and offline streaming in general. Would love feedback, for those interested it is on my github. 
[Kronos: Terminal Music Player](https://github.com/TrevorSatori/Kronos)

![kronos](https://user-images.githubusercontent.com/80570915/218244283-0e1c6f4b-29b7-49b3-a2ec-74d83f30cd88.png)

**Comment by orhun** [comment]
Nice project, watched the repo for new releases!

I'd like to mention, there is also [termusic](https://github.com/tramhao/termusic) which is using [tuirealm](https://crates.io/crates/tuirealm). It might be an inspiration if you are building something similar.
